src/sendResponse.js

```diff
@@ -0,0 +1,20 @@
+const { convertToCase } = require('./convertToCase');
+
+const sendResponse = (errorsCheck, textToConvert, toCase, res) => {
```

Comment:
the function should response to the send response, not handle some cases
Decompose your logic